contracts/utilityToken/UtilityToken.sol

```diff
@@ -153,4 +112,48 @@ contract UtilityToken is ERC20Token {
 
         success_ = true;
     }
+
+
+    /** Internal Functions */
+
+    /**
+     * @notice Sets up the symbol, name, decimals, totalSupply
+     *         and the consensusCogateway address
+     *
+     * @param _symbol Symbol of token.
+     * @param _name Name of token.
+     * @param _decimals Decimal of token.
+     * @param _totalTokenSupply Total token supply.
+     * @param _consensusCogateway ConsensusCogateway contract address.
+     *
+     * @return success_ `true` for a successful setup, `false` otherwise.
+     */
+    function setup(
+        string memory _symbol,
+        string memory _name,
+        uint8 _decimals,
+        uint256 _totalTokenSupply,
+        address _consensusCogateway
+    )
+        internal
+        returns (bool success_)
+    {
+        require(
+            consensusCogateway == address(0),
+            "ConsensusCogateway address is already set."
+        );
+
+        require(
+            _consensusCogateway != address(0),
+            "ConsensusCogateway address should not be zero."
+        );
+
+        tokenSymbol = _symbol;
+        tokenName = _name;
+        tokenDecimals = _decimals;
+        totalTokenSupply = _totalTokenSupply;
+        consensusCogateway = _consensusCogateway;
+
+        success_ = true;
```

Comment:
I don't think that return value is needed.

Reply:
It's an existing implementation. We will need separate ticket to update this.
cc: @jayeshbairagi